docs/UserGuide.md

```diff
@@ -41,138 +31,225 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 > * Items with `...` after them can have multiple instances.
 > * Parameters can be in any order.
 
+### 2.0. Viewing tasks
+
+> * A list of tasks will always be displayed.
```

Comment:
@Rinder5 sorry for keeping asking for more stuff :S can put a `list` here as well. Just list all tasks (referred to in #16)